src/etc/inc/interfaces.inc

```diff
@@ -4539,7 +4539,7 @@ function make_ipv6_64_address($prefix, $suffix)
     return implode(':', $prefix_array);
 }
 
-function interfaces_addresses($interfaces, $as_subnet = false, $ifconfig_details = null)
+function interfaces_addresses($interfaces, $as_subnet = false, $ifconfig_details = null, $include_vips = true)
```

Comment:
I don‘t want to complicate this any more like projects used to. The caller should do the necessary post processing with the help of a return value.